YouTubePlayer/YouTubePlayer/VideoPlayerView.swift

```diff
@@ -315,7 +315,7 @@ public class YouTubePlayerView: UIView, UIWebViewDelegate {
         let url = request.URL
 
         // Check if ytplayer event and, if so, pass to handleJSEvent
-        if url.scheme == "ytplayer" { handleJSEvent(url) }
+        if url!.scheme == "ytplayer" { handleJSEvent(url!) }
```

Comment:
Practically speaking, this is fine, but a safer fix would be the following:
`if let url = url where url.scheme == "ytplayer" { … }`

If for some reason the URL does not exist, the former fix would throw and crash whereas the latter would fail silently.